pages/common/sed.md

```diff
@@ -1,7 +1,7 @@
 # sed
 
 > Edit text in a scriptable manner.
-> More information: <https://man.archlinux.org/man/sed.1>.
+> More information: <https://manned.org/sed>.
```

Comment:
```suggestion
> More information: <https://www.gnu.org/software/sed/manual/sed>.
```

Reply:
I have similar reservations as with `ed` above, but [I get the impression](https://en.wikipedia.org/wiki/Sed#History) that in this case it's more evident that GNU sed is the consensual heir to the original UNIX sed in the Linux world. Is that your understanding as well?

Reply:
In my opinion after such a long time I think it's not that important where the command came from. GNU just documents the command very well here. It has a whole manual about `ed` in contrast to only a man page by `manned.org`.

Reply:
Good point. Will change this later.

Reply:
[Done](https://github.com/tldr-pages/tldr/pull/5790/commits/e6daeecdbfdd99172780d0a073fd1ac778377e01).